src/installation/index.md

```diff
@@ -59,10 +59,12 @@ To build Rust applications for the Espressif chips based on `RISC-V` architectur
 
       This target is currently [Tier 2][rust-lang-book--platform-support-tier2]; note the different flavors of `riscv32` target in Rust covering different [`RISC-V` extensions][wiki-riscv-standard-extensions].
 
-    - For `std` applications, use the target `riscv32imc-esp-espidf`.
-
-      Since this target is currently [Tier 3][rust-lang-book--platform-support-tier3], it does not have pre-built objects distributed through `rustup`, and **it does not need to be installed** as the `no_std` target.
+    - For `std` applications:
 
+      Since this target is currently [Tier 3][rust-lang-book--platform-support-tier3], it does not have pre-built objects distributed through `rustup` and, unlike the `no_std` target, **nothing needs to be installed**. Specify the correct target for your device:
```

Comment:
```suggestion
      Since this target is currently [Tier 3][rust-lang-book--platform-support-tier3], it does not have pre-built objects distributed through `rustup` and, unlike the `no_std` target, **nothing needs to be installed**. Here are the available targets:
```

I would avoid using "specify the correct target" as the user might not know how to do that, and, if they use the template, as we recommend, this will be already specified

Reply:
Done, changed to referencing the rustc book as suggested in your other comment.